test/_vagrant/vagrant_test.go

```diff
@@ -130,6 +134,273 @@ func TestVagrantSetupGuide(t *testing.T) {
 	}
 	t.Fatal("Workflow never got to a complite state or it didn't make it on time (5m)")
 }
+func TestTimeout(t *testing.T) {
```

Comment:
`TestTimeout` does not tell me what you are testing against. What about: `TestActionCorrectlyReachesTimeout`